paddle/fluid/framework/ir/conv3d_bias_mkldnn_fuse_pass.cc

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/framework/ir/conv3d_bias_mkldnn_fuse_pass.h"
+
+REGISTER_PASS(conv3d_bias_mkldnn_fuse_pass,
```

Comment:
You can `REGISTER_PASS(conv3d_bias_mkldnn_fuse_pass)` in `conv_bias_mkldnn_fuse_pass.cc`. And don't add `conv3d_bias_mkldnn_fuse_pass.h/cc`

Reply:
These files was already merged into conv_bias_mkldnn_fuse_pass.cc.